app/assets/javascripts/survey.coffee.erb

```diff
@@ -1,11 +1,22 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+root = exports ? this
```

Comment:
Is this a ternary statement or some coffeescript thing? If so I think we need to be explicit about the false condition too

Reply:
I have removed this unnecessary code thanks!